dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -417,11 +648,9 @@ public void findBySubmitterTest() throws Exception {
                                       .withIssueDate("2016-02-13")
                                       .build();
 
-        // use our admin to retrieve all the workspace by submitter
-        String token = getAuthToken(admin.getEmail(), password);
```

Comment:
Are you still verifying the findBySubmitter method also works for admins to search for an other submitter (since that test seems to be removed here)

Reply:
done, see https://github.com/DSpace/DSpace/pull/2675/commits/5d98bcd9110365c5f91c646fe9a4d8de3eb3cb37#diff-38ba2da3dd74a1027593bbd62d684626R729